nccr-mediality.csl

```diff
@@ -174,7 +174,7 @@
     <group delimiter=", ">
       <choose>
         <if type="book">
-          <text variable="title" form="short" font-style="italic"/>
+          <text variable="title" form="short"/>
```

Comment:
give this, let's simplify the macro. You can reduce it to a single line, i.e. `<text variable="title" form="short"/>`